File: fabric/src/main/java/dev/franckyi/mcfx/fabric/MCFXModFabric.java
package dev.franckyi.mcfx.fabric;

import dev.franckyi.mcfx.fabriclike.MCFXModFabricLike;
import net.fabricmc.api.ModInitializer;

public class MCFXModFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        MCFXModFabricLike.init();
    }
}

File: fabric/src/main/kotlin/dev/franckyi/mcfx/fabric/MCFXModFabric.kt
package dev.franckyi.mcfx.fabric

import dev.franckyi.mcfx.fabriclike.MCFXModFabricLike
import net.fabricmc.api.ClientModInitializer

object MCFXModFabric : ClientModInitializer {
    override fun onInitializeClient() {
        MCFXModFabricLike.init()
    }
}